main.go

```diff
@@ -83,25 +95,28 @@ func main() {
 	// Check dependencies
 	log.Infof("Checking dependencies")
 	log.Printf("Bundler...")
-	if ok, err := rubycommand.IsGemInstalled("bundler", ""); err != nil {
+
+	bundlerVersion, err := getBundlerVersion()
```

Comment:
lets say here we return gems.Version{}, what will happen with the commands of: IsGemInstalled and InstallBundlerCommand? do they handle empty version?

Reply:
`IsGemInstalled` was used previously with always empty string. So I assume that it is working as intended.
`InstallBundlerCommand` should handle also empty strings: https://github.com/bitrise-io/go-utils/blob/master/command/gems/bundler.go#L14 https://github.com/bitrise-io/go-utils/blob/master/command/gems/gemfile_lock.go#L66